htdocs/main.php

```diff
@@ -121,19 +121,6 @@ function tplFromRequest($param)
 $paths = $config->getSetting('paths');
 
 if (!empty($TestName)) {
-    if (file_exists($paths['base'] . "modules/$TestName/js/$TestName.js")) {
-        if (strpos($_SERVER['REQUEST_URI'], "main.php") === false
-            && strcmp($_SERVER['REQUEST_URI'], '/') != 0
-        ) {
-              $tpl_data['test_name_js'] = "js/$TestName.js";
-        } else {
-              $tpl_data['test_name_js'] = "GetJS.php?Module=$TestName";
```

Comment:
This breaks any module that used this line to get the base javascript file for the module